server/models/users.js

```diff
@@ -71,8 +88,42 @@ const getUserModel = (sequelize, Sequelize) => {
       target: "id",
       onUpdate: "CASCADE"
     });
+    User.hasMany(db["Articles"], {
+      foreignKey: "authorId",
+      target: "id",
+      onDelete: "CASCADE"
+    });
+  };
+
+  User.prototype.generateResetToken = function() {
+    this.resetToken = encryptToken();
+    this.save();
+    this.reload();
+    return this.resetToken;
   };
+
+  User.prototype.sendPasswordResetEmail = function(url) {
```

Comment:
I don't believe you should be modifying the prototypes to be implementing the hooks. This seems like a hack. Please re-implement this